### PR TITLE
add viewers field in domain and project dashboard

### PR DESCRIPTION
**Pull request by stat-kwon**
### Category
- [ ] New feature
- [ ] Bug fix
- [x] Improvement
- [ ] Refactor
- [ ] etc

### Description
* add viewers field in domain and project dashboard
* apply view field in test code of domain and project dashboard

### Known issue
* #6

### CODE REVIEW

File: src/spaceone/dashboard/service/domain_dashboard_service.py [hunk 1]
```diff
@@ -111,13 +111,17 @@ def delete(self, params):
         domain_dashboard_vo: DomainDashboard = self.domain_dashboard_mgr.get_domain_dashboard(
             params['domain_dashboard_id'], params['domain_id'])
 
+        if domain_dashboard_vo.viewers == 'PRIVATE' and domain_dashboard_vo.user_id != self.transaction.get_meta(
```
Comment (whdalsrnt) [inline src/spaceone/dashboard/service/domain_dashboard_service.py:114]: Please change the line breaks as follows:
```
 if domain_dashboard_vo.viewers == 'PRIVATE' and \
    domain_dashboard_vo.user_id != self.transaction.get_meta('user_id'):
```

File: src/spaceone/dashboard/service/domain_dashboard_service.py [hunk 2]
```diff
@@ -164,7 +175,7 @@ def delete_version(self, params):
 
         return self.version_mgr.delete_version(domain_dashboard_id, version, domain_id)
 
-    @transaction(append_meta={'authorization.scope': 'DOMAIN_OR_USER'})
+    @transaction(append_meta={'authorization.scope': 'DOMAIN'})
     @check_required(['domain_dashboard_id', 'version', 'domain_id'])
     def revert_version(self, params):
```
Comment (whdalsrnt) [inline src/spaceone/dashboard/service/domain_dashboard_service.py:180]: Requires permission validation logic similar to get and delete methods.
First, search for domain_dashboard_vo and verify the same.

**Review by whdalsrnt — CHANGES_REQUESTED** [review]
As described in the review, `domain_dashboard_service.py` needs to be changed.
Also, change `project_dashboard_service.py` in the same way.

**Comment by stat-kwon** [comment]
I reflected 3 kinds of feedback you said through code review.
* change`ERROR_PERMISSION_DENIED` to `ERROR_PERMISSION_DENIED()`
* change the line breaks ( validation )
* add user validation to `delete_version()`, `revert_version()` and `get_version()`

Please review again. thank you @whdalsrnt

**Review by whdalsrnt — APPROVED** [review]
LGTM

**Pull request merged**